packages/dnslookup/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ameshkov/dnslookup
+TERMUX_PKG_DESCRIPTION="Simple command line utility to make DNS lookups. Supports all known DNS protocols: plain DNS, DoH, DoT, DoQ, DNSCrypt."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="kay9925@outlook.com"
+TERMUX_PKG_VERSION="1.8.1"
+TERMUX_PKG_SRCURL="https://github.com/ameshkov/dnslookup/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=51b9cbc626e091eb7a98bc326ad026f36d95f8b5917f71f13011466fcdddb3f9
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+	termux_setup_golang
+  
+	# This quic-go version cannot be built on go1.20
+	go get -u && go mod tidy
```

Comment:
This is not allowed due to nondeterminism.

Reply:
https://github.com/ameshkov/dnslookup/blob/d3b7532263eac7d939275f2616bbf52e814b1f76/go.mod#L19

As the comment says, this quic-go version cannot be built on go1.20. I can provide a patch that updates dependencies to replace it.
[0001-update-dependencies.patch](https://github.com/termux/termux-packages/files/10920640/0001-update-dependencies.patch)

Reply:
Good job!